### DISCUSSION TITLE
Let us change the order of repositories in lists

### DISCUSSION THREAD

**Discussion by CanePlayz** [community/community#45668] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It'd be really really useful to be able to customize the order in which repositories in lists appear. Filters could also be added to the new lists feature.

**Comment by jge162** [comment]
Thank you for your post @CanePlayz, are you just trying to filter the stuff you see in the repo list? You can use the column 'headers' or using the "Sort by" dropdown in GitHub changes the order of the repositories in the list.

See the three drop_downs can be used for customizing which repositories in lists appear. 

<img width="636" alt="Screen Shot 2023-01-28 at 3 38 50 PM" src="https://user-images.githubusercontent.com/31228460/215296047-ca124a6f-bd0e-49b9-be52-6334fb2998b6.png">

**Reply by CanePlayz** [reply]
Oh, I meant the custom lists which are in beta, not the list of all of your repositories. Should have made that more clear. 😅 And I wasn't talking about filters (although those could be introduced as well), I was rather talking about the order in which they appear in the list.

**Reply by jge162** [reply]
Gotchu. Thanks for post. Have a great weekend.

**Reply by CanePlayz** [reply]
Thanks, you too :)

**Comment by gutwrinch** [comment]
I think that is a great idea although I don't have the experience to know exactly what you're talking about when you say move the order it would be a great idea and opportunity for someone to take me under their wing and show me everything I'm doing wrong because it's a lot

**Comment by JuampaVLB** [comment]
I think that is a great idea although I don't have the experience to know exactly what you're talking about when you say move the order it would be a great idea and opportunity for someone to take me under their wing and show me everything I'm doing wrong because it's a lot

**Comment by Monicaann0o** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> It'd be really really useful to be able to customize the order in which repositories in lists appear. Filters could also be added to the new lists feature.

Respiratory 0(dove)

**Comment by stez-mind** [comment]
Indeed, this would be very useful.

I've started to organize my stars into lists, but once I select a list it would be great to sort the repos inside it in a similar fashion as the global starred list, e.g. sorting by last updated, number of stars, last added, etc.

**Reply by stez-mind** [reply]
Another way to implement the same feature would be to add a "list" option as a filter in the global stars view.

This would be perfect since you'll have also the other filtering & keyword options already available.

**Reply by gutwrinch** [reply]
Do it
